community/nload/build

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh -e
+
+./configure \
+	--prefix=/usr
+make DESTDIR="$1" install
```

Comment:
Indentation is also four spaces. It should look the same as the suggested fix I sent earlier.